kubernetes/conf.yaml.example

```diff
@@ -11,6 +11,9 @@ instances:
   # reach the kubelet and cadvisor APIs unless the environment variable
   # KUBERNETES_KUBELET_HOST is found (if your node names can be resolved by pods, we recommend you set this variable to spec.nodeName through the downward API).
   #
+  # To enable Kube State metrics, please refer to kubernetes_states integration. For more information,
```

Comment:
The doco at http://docs.datadoghq.com/integrations/kubernetes/#kubernetes-state-metrics uses "Kubernetes State Metrics" and "kube-state-metrics". I'd suggest using those terms instead of "Kube State metrics" and "kubernetes_states".